tests/Backend/CommonPart2/SimpleAliasTest.php

```diff
@@ -879,4 +879,77 @@ public function testTableAliasFromDevBranchBucketCannotBeCreated()
 
         $this->assertFalse($this->_client->tableExists($aliasTableId));
     }
+
+    public function testTableAliasInDevBranchBucketCannotBeCreated()
+    {
+        $metadataProvider = 'system';
+        $metadataKey = 'KBC.createdBy.branch.id';
+
+        $importFile = __DIR__ . '/../../_data/languages.csv';
+
+        $bucketId = $this->getTestBucketId();
+
+        // create and import data into source table
+        $sourceTableId = $this->_client->createTable(
+            $bucketId,
+            'languages',
+            new CsvFile($importFile),
+            array(
+                'primaryKey' => 'id'
+            )
+        );
+
+        $metadata = new Metadata($this->_client);
+
+        // check that validation ignores table/columns metadata
+        $metadata->postColumnMetadata(
+            sprintf('%s.%s', $sourceTableId, 'id'),
```

Comment:
tím, že ten source nastavíš jako dev, tak netestuješ, že to nejde z "normálního do dev" ale že to nejde "z dev do dev". Myslím, že bys to měl upravit tak, že se ten alias vytváří třeba z IN bucketu (který nebude mít metadata) do OUT bucketu (který bude mít metadata).

Reply:
sry, teda napsal jsem to na špatný řádek. Mělo to být níž, ale tam ten $bucketId je furt ten bucket ve kterém je $sourceTableId. Tím pádem sourceBucket === destinationBucket

Reply:
Otestoval jsem to tak, že jsem přehodil v connection test bucketu o 3 řádky níž. Test spadnul i když ten kód je ekvivalentní.